docs/samples/specification/management/generated/setup.py

```diff
@@ -19,7 +19,9 @@
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
 
-REQUIRES = ["msrest>=0.6.21", "azure-core<2.0.0,>=1.20.1", "azure-mgmt-core<2.0.0,>=1.2.1"]
+REQUIRES = [
+    "msrest>=0.6.21",
+    "azure-core<2.0.0,>=1.20.1",    "azure-mgmt-core>=1.3.0,<2.0.0"]
```

Comment:
this is gernating quite weirdly, I left a comment about potential fixes in the template